app/views/layouts/_header.html.erb

```diff
@@ -2,21 +2,8 @@
   <ul>
     <li class="mobile-hidden"><%= link_to 'Home', root_path %></li>
     <% if current_user.present? %>
-      <li><%= link_to 'My Movies', movies_path %></li>
-      <li><%= link_to 'My Lists', user_lists_path(current_user) %></li>
```

Comment:
"My Movies" now lives on the "My Lists" page and the navbar item for lists is now called "My Movies".

Reply:
real nice improvement! 🚀